apps/src/javalab/javabuilderExceptionHandler.js

```diff
@@ -121,7 +127,11 @@ export function handleException(exceptionDetails, callback) {
       break;
 
     default:
-      error = msg.unknownError({type, connectionId});
+      if (fallbackMessage) {
+        error = fallbackMessage;
+      } else {
+        error = msg.unknownError({type, connectionId});
+      }
```

Comment:
nit: simplify to `error = fallbackMessage || msg.unknownError(...)`?

Reply:
Thanks!